Crossover patcher/pl.lproj/Localizable.strings

```diff
@@ -1,98 +1,127 @@
-"CXPatcherName" = "CXPatcher";
-"DisclaimerPleaseNoteLabelText" = "Uwaga:";
-"DisclaimerText" = "To jest nieoficjalny program umożliwiający łatanie programu CrossOver. CodeWeavers nie jest w żaden sposób związane z pracami i rozwojem tego programu. Pomimo przeprowadzonych testów, to oprogramowanie może spowodować, że CrossOver będzie działał w nieprawidłowy sposób. KORZYSTASZ NA WŁASNE RYZYKO. Korzystanie z tego oprogramowania narusza warunki oficjalnego wsparcia ze strony CodeWeavers.
+"waitFor" = "Czekam na przeczytanie zastrzeżenia i potwierdzenie";
+"confirmation" = "Wpisz w polu poniżej: 'Nie będę prosić CodeWeavers o wsparcie ani zwrot pieniędzy'";
+"confirmationValue" = "Nie będę prosić CodeWeavers o wsparcie ani zwrot pieniędzy";
 
-Jeśli napotkasz jakiekolwiek problemy z instalacją CrossOver, możesz pobrać nową kopię CrossOver ze strony CodeWeavers.";
-"XtLibsButtonText" = "Zlokalizuj folder DXMT";
-"XtLibsLabel" = "Zainstaluj zewnętrzny framework";
-"XtLibsToggle" = "Zainstaluj DXMT";
-"XtLibsLocateText" = "Zlokalizuj folder DXMT za pomocą przycisku poniżej:";
-"XtLibsPathLabelText" = "Ścieżka do folderu zewnętrznych zasobów";
-"XtLibsDisclaimerText" = "OSTRZEŻENIE: To wersja Alpha — może być niestabilna i spowodować awarię systemu. Używaj na własne ryzyko. Po zastosowaniu łatki MUSISZ UTWORZYĆ NOWĄ BUTELKĘ DLA DXMT.";
-"GPTKToggle" = "Integracja D3DMetal (GPTK)";
-"ForcePatch" = "Wymuś łatanie";
+/* --- Main Navigation & Buttons --- */
+"AgreeAndProceedButtonText" = "Zgadzam się i kontynuuj";
+"continue" = "Kontynuuj";
+"Options" = "Opcje zaawansowane";
+"optionsButtonText" = "Opcje zaawansowane";
+"toolsButtonText" = "Narzędzia";
+"InstructionsButtonText" = "Instrukcja";
+
+/* --- Instructions --- */
 "Instructions" = "Instrukcja";
 "InstructionsAskWhatDo" = "Co umożliwia ten program?";
-"InstructionsButtonText" = "Instrukcja";
 "InstructionsFunctionDescription" = "Ten program modyfikuje Twoją instalację CrossOver, aby używała zaktualizowanych i dostosowanych wersji DXVK i MoltenVK, co w niektórych przypadkach poprawia kompatybilność lub wydajność gier.";
 "InstructionsText" = "Upewnij się, że masz niezmodyfikowaną wersję CrossOver pobraną ze strony swojego konta w CodeWeavers. Upewnij się, że CrossOver został zarejestrowany i uruchomiony przynajmniej raz, aby najnowsza wersja DXVK została poprawnie aktywowana. Może być konieczne przełączenie DXVK na wyłączony i ponownie włączony w CrossOver. Jeśli z jakiegoś powodu Twoja instalacja CrossOver stała się nieużywalna, możesz ją ponownie pobrać ze strony CodeWeavers. Ten program nie wprowadza trwałych zmian do Twoich butelek CrossOver.";
-"PatchStatusAlreadyPatched" = "Ta wersja CrossOver.app została już wcześniej załatana!";
-"PatchStatusError" = "Ta wersja CrossOver nie jest obsługiwana. Użyj CrossOver %@ ze strony CodeWeavers.";
+"CrossOverDownloadURLText" = "Pobierz CrossOver z oficjalnej strony CodeWeavers";
```

Comment:
This is a duplicate key i guess you added it but you forgot to delete the one on line 14